packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -793,16 +798,21 @@ class TextSelectionOverlay {
     }
 
     final TextSelection newSelection;
+    final bool dragStartSelectionNormalized = _dragStartSelection!.extentOffset >= _dragStartSelection!.baseOffset;
```

Comment:
Could you use _dragStartSelection.isNormalized? Here and below.

Reply:
`isNormalized` seemed to behave a bit strangely to me, maybe `TextSelection` should have an override for it, but the default implementation checks `end >= start`, which will always be `true` with regards to a `TextSelection`, since `TextSelection` passes normalized values to `TextRange`. What do you think?

Reply:
Ah ok, keeping it as-is is fine then. I think I've run into the same thing before. Maybe add a comment to avoid someone else coming in and moving this to _dragStartSelection.isNormalized if you think it might help.